ops/model.py

```diff
@@ -126,6 +126,7 @@
 
 
 StrOrPath = typing.Union[str, Path]
+StrOrPurePath = typing.Union[str, PurePath]
```

Comment:
Let's remove this and just spell out `Union[str, PurePath]` wherever it's used (you could make this fix for `StrOrPath` if you want, but not a requirement for this PR). I think it's better documentation, as we don't have to document (or have them look up) what `StrOrPath` means.

Also, let's drop the `typing.` prefix as `Union` is imported directly.